lib/matplotlib/text.py

```diff
@@ -130,6 +130,7 @@ def __init__(self,
                  rotation_mode=None,
                  usetex=None,          # defaults to rcParams['text.usetex']
                  wrap=False,
+                 rotation_transformed=False,
```

Comment:
I'm struggling a bit with the name, maybe `transform_rotates_text`? Better suggestions welcome.